scripts/release/asset_publish.py

```diff
@@ -267,7 +267,9 @@ def _str2bool(v: str) -> bool:
         assets_by_type[asset.type.value].append(asset)
 
     # mlclient for the registry
-    mlclient = MLClient(DefaultAzureCredential(), subscription_id=subscription_id, registry_name=registry_name)
```

Comment:
You should be able to use AzureCliCredential() instead. What error are you running into?

Reply:
Got this issue: 

2023-03-07 05:37:10.480808: DefaultAzureCredential failed to retrieve a token from the included credentials.

Reply:
It passes locally, but not in ev2 agent. Default credential also seemed to work here: [release](https://msdata.visualstudio.com/Vienna/_releaseProgress?_a=release-pipeline-progress&releaseId=834017)

Reply:
Please share the source of the log message you shared above via Teams. The publish_assets.sh script runs `az login` before this script is called and this is what allows DefaultAzureCredential to work.

Reply:
DefaultCredential is working. Updated PR